forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BaseDeviceBindingTest.java

```diff
@@ -47,6 +52,9 @@ public abstract class BaseDeviceBindingTest {
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
+    @Rule
+    public RetryTestRule retry = new RetryTestRule(3);
```

Comment:
we are introducing retry now ? test retries can be a blessing or a curse. we need to keep an eye on how much we do in all the test

Reply:
Yes, I agree... I removed this retry from the device-binding tests...